tokio-postgres/src/config.rs

```diff
@@ -326,8 +336,8 @@ impl Config {
 
     /// Gets the connection timeout, if one has been set with the
     /// `connect_timeout` method.
-    pub fn get_connect_timeout(&self) -> Option<&Duration> {
-        self.connect_timeout.as_ref()
+    pub fn get_connect_timeout(&self) -> Option<Duration> {
+        self.connect_timeout.as_ref().copied()
```

Comment:
```suggestion
        self.connect_timeout
```

Reply:
Fixed in 5551b3e6cdd81c18d5ffb13ee5478fcd0e551475